README.md

```diff
@@ -73,6 +73,19 @@ To build all pre-configured images run:
 $ .ci/build
 ```
 
+To update pre-configured images supported k8s versions run then commit:
+```bash
+$ gardenctl target garden live
+$ gardenctl kubectl get cloudprofile -o json | jq -r '.items[].spec.kubernetes.versions[] | select(.classification=="supported") | .version' |sort -r |uniq > .ci/k8s_versions
+$ .ci/build_pipeline_definitions
```

Comment:
```suggestion
gardenctl target garden <landscape>
gardenctl kubectl get cloudprofile -o json | jq -r '.items[].spec.kubernetes.versions[] | select(.classification=="supported") | .version' |sort -r |uniq > .ci/k8s_versions
.ci/build_pipeline_definitions
```

Reply:
The command to update the `k8s_versions` file should make sure that only the latest patch version of a minor version is added.
e.g. the versions are
```
1.20.5
1.19.9
1.19.8
```

only 
```
1.20.5
1.19.9
```
should be added

Reply:
What about the deprecated minor versions? You can still create clusters for them